theme/index.pug

```diff
@@ -1,4 +1,4 @@
-- ts.sort((a, b) => a.name.localeCompare(b.name))
+- var toc = Object.keys(ts).sort((a, b) => a.localeCompare(b))
```

Comment:
just that you changed this array to an object in this PR and now you're converting back to an array.